### PR TITLE
todo project for partner review

### PR DESCRIPTION
**Pull request by blakbox23**
This is my ToDo project built as per the Microverse requirements.
Please review it for me.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+const addToTasks = (e) => {
+  e.preventDefault();
+  tasks.push({
+    description: taskDescription.value,
+    completed: false,
+    index: tasks.length,
+  });
+  addToLocalStorage();
+  window.location.reload();
```
Comment (vikitaotiz) [inline src/crud.js]: Reloading the browser after every addition or deletion of todo is not a good user experience. I think you should have used target parentNode to grab onto the ui elements to make it easier to make the interface reactive.

**Review by vikitaotiz — CHANGES_REQUESTED** [review]
Reloading the browser after every addition or deletion of todo is not a good user experience. I think you should have used target parentNode to grab onto the ui elements to make it easier to make the interface reactive.

**Comment by blakbox23** [comment]
I've fixed the KISS, DRY, and YAGNI errors. I have also reduced the number of reloads by one and now don't need to reload to add a task. any help on removing reloads while deleting would be highly appreciated.

**Review by vikitaotiz — APPROVED** [review]
## Approved 🥳
Keep up the good work!!!

**Pull request merged**